Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/ShellTitleView.cs

```diff
@@ -58,7 +63,19 @@ ContentPage createContentPage(string titleView)
 		}
 
 
-#if UITEST && (__IOS__ || __ANDROID__)
+#if UITEST
+
+		[Test]
+		public void TitleViewPositionsCorrectly()
+		{
+			var titleView = RunningApp.WaitForElement("TitleViewId")[0].Rect;
+			var topTab = RunningApp.WaitForElement("page 2")[0].Rect;
+
+			var titleViewBottom = titleView.Y + titleView.Height;
+			var topTabTop = topTab.Y;
+
+			Assert.GreaterOrEqual(topTabTop, titleViewBottom, "Title View is incorrectly positioned behind tabs");
+		}
```

Comment:
What do you mean by "tabs" and "topTab" here?

Reply:
On the UI test there are tabs at the top of the screen. The best way I could find to test if the TitleView is positioned correctly is to validate that the coordinates of the TitleView don't intersect with the Tabs since the Tabs are directly below the titleview

Reply:
ahh ok, makes sense. Thanks!